docs/source/getting_started_facedancer.rst

```diff
@@ -2,6 +2,9 @@
 Using Cynthion with Facedancer
 ==============================
 
+Together with `Facedancer <https://github.com/greatscottgadgets/facedancer/>`__,
```

Comment:
For consistency, I think it would be better to link to the Facedancer docs here.